lib/browser/rpc-server.js

```diff
@@ -90,18 +89,21 @@ const getPreloadScript = async function (preloadPath) {
   return { preloadPath, preloadSrc, preloadError }
 }
 
-if (features.isExtensionsEnabled()) {
-  ipcMainUtils.handleSync('ELECTRON_GET_CONTENT_SCRIPTS', () => [])
```

Comment:
Minor nit: this was split into two calls to `handleSync` because the idea was that eventually, with extensions enabled, this handler would go away, and with it this branch of the `if` statement.

Reply:
I was wondering why you did it like that.

Reply:
reverted